### PR TITLE
Generate seed data and initial seeding to mysql database review

### PR DESCRIPTION
**Pull request by lison12**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -23,6 +23,19 @@ app.get('/listings/:id/reviews', (request, response) => {
   });
 });
 
+
+app.get('/listings/:id/listings', (request, response) => {
```
Comment (zouboyun) [inline server/server.js]: would it be sufficient to just have '/listings/:id' ?

Comment (lison12) [reply server/server.js]: This is to get the listings data

File: client/src/components/reviews.jsx [hunk 2]
```diff
@@ -1,36 +1,72 @@
...
+  }
+
+
   render() {
+    // console.log(this.state)
     return (
       <div>
         <div>
-        <span className="FiveStar__FiveStarTop-hLclkH cJfAex" width="NaN"><span className="FiveStar__SingleStar-Vpwis bUXsVT">★</span><span class="FiveStar__SingleStar-Vpwis bUXsVT">★</span><span class="FiveStar__SingleStar-Vpwis bUXsVT">★</span><span class="FiveStar__SingleStar-Vpwis bUXsVT">★</span><span class="FiveStar__SingleStar-Vpwis bUXsVT">★</span></span>
+        <span className="FiveStar__FiveStarTop-hLclkH cJfAex" width="NaN"><span className="FiveStar__SingleStar-Vpwis bUXsVT">★</span><span className="FiveStar__SingleStar-Vpwis bUXsVT">★</span><span className="FiveStar__SingleStar-Vpwis bUXsVT">★</span><span className="FiveStar__SingleStar-Vpwis bUXsVT">★</span><span className="FiveStar__SingleStar-Vpwis bUXsVT">★</span></span>
```
[the remark this answers is not in the document]
Comment (lison12) [reply client/src/components/reviews.jsx]: I removed these stars div

File: client/src/components/reviews.jsx [hunk 3]
```diff
@@ -1,36 +1,72 @@
...
     this.state = {
+      list_data: [],
+      review_data: [],
       satisfied: 'I love this product!!',
       unhappy: 'Arrived late and damaged :('
     };
   }
 
+  componentDidMount() {
+    const id = 33;
```
[the remark this answers is not in the document]
Comment (lison12) [reply client/src/components/reviews.jsx]: I fixed the id to get the correct param from the url

**Pull request merged**